<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgMills</class>
 <widget class="QDialog" name="CQDlgMills">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mill Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="qqEnableAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Enable all mills</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Main properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqExplicitHandling">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Explicit handling</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqAdjustVolume">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show volume parameters</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="qqEntityToCut">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Entity to cut</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Visual properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqPassiveVolumeColor">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust volume color (passive)</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqActiveVolumeColor">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust volume color (active)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqBaseSize">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Size [m]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqEnableAll</tabstop>
  <tabstop>qqExplicitHandling</tabstop>
  <tabstop>qqEntityToCut</tabstop>
  <tabstop>qqAdjustVolume</tabstop>
  <tabstop>qqBaseSize</tabstop>
  <tabstop>qqActiveVolumeColor</tabstop>
  <tabstop>qqPassiveVolumeColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
